src/Pods/Whatsit/Block.php

```diff
@@ -43,18 +43,78 @@ public function get_block_args() {
 			'category'        => $category,
 			'icon'            => $this->get_arg( 'icon', 'align-right' ),
 			'keywords'        => Tribe__Utils__Array::list_to_array( $this->get_arg( 'keywords', 'pods' ) ),
-			'supports'        => $this->get_arg( 'supports', [
-				'html' => false,
-			] ),
+			'supports'        => $this->get_arg( 'supports', [] ),
 			'editor_script'   => $this->get_arg( 'editor_script', 'pods-blocks-api' ),
 			'fields'          => $this->get_block_fields(),
-			'attributes'      => [
-				'className' => [
-					'type' => 'string',
-				],
+			'attributes'      => $this->get_arg( 'attributes', [] ),
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: bc05f300048bbd36c2c6e7564fae8749</sub>